corehq/blobs/migrate.py

```diff
@@ -240,6 +241,15 @@ def migrate(self, *args, **kw):
         hqmedia.CommCareVideo,
         hqmedia.CommCareMultimedia,
     ], CouchAttachmentMigrator),
+    Migrator("xforms", [
+        xform.XFormInstance,
+        ("XFormInstance-Deleted", xform.XFormInstance),
+        xform.XFormArchived,
+        xform.XFormDeprecated,
+        xform.XFormDuplicate,
+        xform.XFormError,
+        xform.SubmissionErrorLog,
```

Comment:
Note to self: looks like [`HQSubmission`](https://github.com/dimagi/commcare-hq/pull/12333/files#diff-93c384a6794b8c149c3fe4243e26957aR29) is missing from this list. Not sure if that's important.